front/src/applications/stdcmV2/components/StdcmInputVia.tsx

```diff
@@ -20,27 +24,68 @@ const StdcmInputVia = ({
     pathStep.stopFor ? `${ISO8601Duration2sec(pathStep.stopFor) / 60}` : ''
   );
 
+  const [showWarning, setShowWarning] = useState(false);
```

Comment:
Maybe we could rather have a const like this and avoid having a state for the warning

```suggestion
  const stopWarning = stopType === StdcmStopTypes.DRIVER_SWITCH && Number(pathStepStopTime) < 3;
```

Reply:
fixed